cluster_tools/__init__.py

```diff
@@ -184,6 +184,39 @@ def __init__(self, **kwargs):
         WrappedProcessPoolExecutor.__init__(self, **kwargs)
 
 
+class DebugSequentialExecutor(SequentialExecutor):
+    def submit(self, *args, **kwargs):
```

Comment:
Please add a comment outlining what differentiates this Executor from the SequentialExecutor :)

Reply:
good point! i added some docstrings now.